script/setup/install-cni

```diff
@@ -21,7 +21,19 @@
 #
 set -eu -o pipefail
 
-CNI_COMMIT=${1:-$(grep containernetworking/plugins "$GOPATH"/src/github.com/containerd/containerd/go.mod | awk '{print $2}')}
+GOMODEnv=`go env |grep 'GOMOD=' | awk -F '=' '{print $2}' | sed 's/\"//g'`
```

Comment:
`go env | grep -Po '(?<=GOMOD=")(.*)(?=")'` will reduce the piping at the expense of requiring a grep with `-P`

Reply:
I don't think any grep-ing would be needed; `go env` takes arguments, so this would work;

```bash
go env GOMOD
/Users/thajeztah/go/src/github.com/containerd/containerd/go.mod
```